templates/metadata/metaschema/metaschema.1.schema.json

```diff
@@ -47,6 +47,14 @@
             }
           }
         },
+        "include_info_sections": {
+          "type": "boolean",
```

Comment:
feels like we should reverse these properties so the generated version matches the template version